doc/subsystems/test/ztest.rst

```diff
@@ -82,6 +82,55 @@ The above test is then enabled as part of the testsuite using::
     ztest_unit_test(test_assert)
 
 
+Listing Tests
+=============
+
+Tests (test projects) in the Zephyr tree consist of many testcases that run as
+part of a project and test similar functionality, for example an API or a
+feature. The ``sanitycheck`` script can parse the the testcases in all
+test project or a subset of them and can generate reports on a more granular
+level, i.e. if cases have passed or failed or if they were blocked or skipped.
+
+To know about skipped or blocked test cases we do parse the case names from the
```

Comment:
How about this:

    Sanitycheck parses the source files looking for test case names, so you
    can list all kernel test cases, for example, by entering::